cernopendata/modules/fixtures/data/docs/cod-portal-upgrade-31-1-2024/cod-portal-upgrade-31-1-2024.md

```diff
@@ -0,0 +1,14 @@
+A new version of the CERN Open Data Repository has been deployed in January 2024.
+Some of the highlights of this release are:
+
+* Change of technologies, from Angular to [ReactJS](https://react.dev/) and from Bootstrap to [Semantic UI](https://semantic-ui.com/)
```

Comment:
Official name is React, not ReactJS.  And I don't think it is necessary to mention Semantic UI, the open data users don't really care about whether we use Semantic UI or Material UI.  We could even remove Angular/React and just say "Improvements to the web interface...".

Reply:
For the react vs reactjs, it seems to be known by both names. 
Regarding removing also react/semantic, we could indeed remove all the highlights and just leave one-liner saying that we have improved the web interface.  In my opinion, that would be too simplistic, and giving more information might be appreciated by the user.